bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx135.html

```diff
@@ -0,0 +1,45 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_135') }}
+{% endblock %}
+
+{% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=wnp135-shortcuts' %}
+
+{% if LANG == 'de' %}
+  {% set main_title = 'Ein Zoom auf die Funktionen von Firefox' %}
+  {% set main_tagline = 'Navigiere dich mit Firefox schnell durchs Web und nutze <strong>cmd/Strg +</strong> und <strong>cmd/Strg -</strong>, um unkompliziert Inhalte zu vergrößern oder zu verkleinern.' %}
+  {% set main_cta = 'Entdecke weitere Tastenkombinationen' %}
+{% elif LANG == 'fr' %}
+  {% set main_title = 'Zoomez sur les fonctionnalités de Firefox' %}
+  {% set main_tagline = 'Naviguez sur Firefox rapidement avec <strong>cmd/ctrl +</strong> et <strong>cmd/ctrl -</strong> pour facilement redimensionner le contenu d’une page web.' %}
+  {% set main_cta = 'Découvrir plus de raccourcis' %}
+{% else %}
+  {% set main_title = 'Zoom in to Firefox functions' %}
+  {% set main_tagline = 'Navigate Firefox quickly and use <strong>cmd/ctrl +</strong> and <strong>cmd/ctrl -</strong> to easily resize content, making it larger or smaller.' %}
+  {% set main_cta = 'Find more shortcuts' %}
+{% endif %}
+
+{% set main_cta_link = 'https://support.mozilla.org/kb/keyboard-shortcuts-perform-firefox-tasks-quickly?'  + utm_params %}
+
+{% block wnp_content %}
```

Comment:
is it possible to make this localizable & use the custom translations, or should those be separate templates